go/client/cmd_ctl_stop_osx.go

```diff
@@ -94,16 +94,25 @@ func ctlStop(g *libkb.GlobalContext, components map[string]bool) error {
 			errs = append(errs, err)
 		}
 	}
+	if err := rpcStop(g); err != nil {
+		g.Log.Info("ctlStop: expected error when trying to shutdown service post-uninstall: %s", err)
+	} else {
+		g.Log.Info("ctlStop: no error when shutting down service post-uninstall")
```

Comment:
```suggestion
		g.Log.Debug("ctlStop: no error when shutting down service post-uninstall")
```

Reply:
Given how the app behaves when the service gets orphaned from launchd, it seems like it is listening to RPCs. The app starts and works, but it is just not able to be controlled by launchd. The point of this is to attempt an RPC on it (to kill it), when it shouldn't even be running to make sure it is gone. 

The other option was to try to just straight kill it, but there were concerns it would take out other keybase processes that aren't the one supposedly controlled by launchd.

Reply:
Got it. I guess the remaining concern is if the service is running but somehow frozen/not accepting RPCs.

Reply:
At least the app will be completely dead in that case, instead of the version mismatch.